extensions/tika/runtime/src/main/java/io/quarkus/tika/Metadata.java

```diff
@@ -0,0 +1,28 @@
+package io.quarkus.tika;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+public class Metadata {
```

Comment:
I would get rid of that one and expose `TikaMetadata`. I don't see any value in having an additional abstraction layer.

Reply:
Originally I started with this one I think, but Tika [Metadata](https://github.com/apache/tika/blob/master/tika-core/src/main/java/org/apache/tika/metadata/Metadata.java) interface is a bit verbose and somewhat complex (arrays, etc), and it is **publicly modifiable** so my plan was expose it via a simple API sufficient to get the String metadata properties.
Does it sound reasonable ?

Reply:
I still think that this one should be named `TikaMetadata` to be consistent with `TikaContent`.

As for being reasonable, yes. I'm not a big fan of instantiating new objects in a getter but maybe it makes sense to not do it for all the extracted metadata, considering you would probably only use part of it.